RELEASE.md

```diff
@@ -3,7 +3,7 @@
 #### Bump version
 
 1. Open a branch for the release
-    - `git checkout -b releases/1.3.3rc1`
+    - `git checkout -b releases/1.3.3.1rc1`
```

Comment:
I don't think we have to update this file with each release. Let it be hardcoded to the same value as the placeholder (1.0.0a ?)

Reply:
Adam,
I checked the history of this file and has been modified with releases.
If not required will make it back to 1.0.0a

Reply:
Yes, my recommendation is that you keep it unchanged with releases. It's just busy work.

Reply:
Sure Adam